twoopstracker/twoops/views.py

```diff
@@ -407,6 +407,16 @@ def get(self, request, *args, **kwargs):
 
         return self.retrieve(request, *args, **kwargs)
 
+    def delete(self, request, *args, **kwargs):
+        accounts = self.request.data.get("accounts", [])
```

Comment:
Unless there is a reason, lets standardize use of [`xxx[]`](https://github.com/CodeForAfrica/TwoopsTracker/blob/3ed5bb127098fb26a0434b64a54dcdd05be172ba/twoopstracker/twoops/views.py#L424) for passing list of query params (`xxx[]=1&xxx[]=2`).

Reply:
I see. 
But, given that this is a `delete` request just like `post` is it a good idea to have query parameters in the URL? I think we had such a discussion sometimes back. 
Also I feel like the URLs might get extremely long

Reply:
1. The id of whatever you're deleting or updating is **always** on the URL. **Always**. This is because there is nothing wrong with id being in the URL or even logged somewhere. Content however e.g. user password, secret back account number, etc., shouldn't be passed via a URL because you don't wanted them cached/logged anywhere.
2. A URL can be [a few thousands character long](https://stackoverflow.com/a/417184) without any issue. This is also an automatically generated URL which a user will never have to type or even see.

Reply:
> The id of whatever you're deleting or updating is always on the URL. Always.

I agree. But it's usually there as a `path` not a `query` parameter but I think this too can work.
For the URL bit, assuming each account_id is 10 characters long, If there should be an issue then we will start getting them as soon as someone decides to delete 200 accounts from their Lists.

Reply:
Still not sure I see the issue... I have 400 to delete, I send 2 requests with 200 each.

To me and most APIs I know, DELETE, PUT, PATH, GET, always specify the target resource to operate on in the URL (path or query depends on approach taken e.g. we can move accounts DELETE to /accounts and there it will be on path and not query, etc.).

... but as always, may be a concrete example can help me see what you see.

Reply:
After reading around I think they’re both okay approaches but the best practice would have been sending delete requests to
`lists/<list_id>/<account_id>`